src/components/Highlight/index.tsx

```diff
@@ -25,18 +29,41 @@ const Highlighted = styled.span`
  * Highlight component is meant to be a simple component to display text with highlighted search.
  * Component will render a SPAN or series of SPAN with the content and highlights
  */
-export const Highlight: React.FC<HighlightProps> = ({ text, highlight, ignoreCase = true, theme }) => {
+export const Highlight: React.FC<HighlightProps> = ({ text, matches, highlight, ignoreCase = true, theme }) => {
+  if (!matches && !highlight) {
```

Comment:
```suggestion
  if (!matches && undefined === highlight) {
```

You've broken this [invariant](https://github.com/ZEISS/precise-ui/pull/208/files#diff-45bf95c2583e477c07465a08c647e183R56-R58).

Reply:
made the change, why exactly that way only for highlight?

Reply:
nvm